src/processor/operator/persistent/export_db.cpp

```diff
@@ -29,7 +29,7 @@ static void writeStringStreamToFile(
 static void writeCopyStatement(
     stringstream& ss, std::string tableName, ReaderConfig* boundFileInfo) {
     ss << "COPY ";
-    ss << tableName << " FROM \"" << boundFileInfo->filePaths[0] << "/" << tableName;
+    ss << tableName << " FROM \"" << tableName;
```

Comment:
I would suggset using stringFormat for better readability : ss << common::stringFormat("copy from {} {}", tableName, fileTypeStr, csvConfig);